src/roster/views.py

```diff
@@ -1525,7 +1609,7 @@ def get(self, request, *args, **kwargs):
             }
 
         except ClusiveUser.DoesNotExist:
-            messages.error(request, f"Student '{kwargs['username']}' not in this class ({period.name})")
+            messages.error(request, "Student '" + kwargs['username'] + "' not in this class (" + period.name + ")")
```

Comment:
It's because I didn't know there is a `f-string` technique. I thought `f` is a typo and removed it. Then the message was not printed out properly so I had to change it to regular string concatenation.

Reply:
Oh.  If you still don't know, f-strings work like back ticks in javascript.  For example, using the same variable names, in JS it would be:
```
console.error(`Student '${kwargs['username']}' not in this class (${period.name})`);
```